ios/FluentUI/Tooltip/TooltipTokenSet.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `Tooltip` control.
+public class TooltipTokenSet: ControlTokenSet<TooltipTokenSet.Tokens> {
+    public enum Tokens: TokenSetKey {
+        /// The color of the body of the tooltip.
+        case tooltipColor
+
+        /// The color of the text within the tooltip.
+        case textColor
+
+        /// The  information for the tooltip's shadow.
+        case shadowInfo
+
+        /// The radius for the corners of the tooltip.
+        case backgroundCornerRadius
+
+        /// The TextStyle of the message label.
+        case messageLabelTextStyle
+
+        /// The TextStyle of the title label.
+        case titleLabelTextStyle
+
+        /// The maximum width of the tooltip.
+        case maximumWidth
+
+        /// The height of the arrow of the tooltip.
+        case arrowHeight
+
+        /// The width of the arrow of the tooltip.
+        case arrowWidth
+    }
+
+    init() {
+        super.init { token, theme in
+            switch token {
+            case .tooltipColor:
+                return .dynamicColor { DynamicColor(light: ColorValue(r: 33.0 / 255.0,
```

Comment:
nit: if you do these floating math it is little bit inaccurate compare to hex itself. we can't use DynamicColor(light: ColorValue(0xFFFFFF) ? like you did in shimmer?

Reply:
what about the alpha?

Reply:
argh. great point!